packages/client/src/utils/generateInFolder.ts

```diff
@@ -70,7 +74,11 @@ export async function generateInFolder({
         overwrite: true,
       })
     } else {
-      await getPackedPackage('@prisma/client', outputDir)
+      await getPackedPackage(
+        '@prisma/client',
+        outputDir,
+        process.cwd().endsWith('output-directory') ? path.join(__dirname, '..', '..') : undefined,
```

Comment:
Does it mean that it actually depends on the `relativeOutputDir` maybe?
Either way, I'd prefer to not hardcode the specific test name here.

Reply:
Yeah same, I'll keep trying things I guess.

Reply:
I think my new implementation is better?